test/channel_test.js

```diff
@@ -726,15 +726,11 @@ describe('on', () => {
   it('"*" bind all events', () => {
     const spy = sinon.spy()
 
-    channel.trigger('INSERT', {}, defaultRef)
-    assert.ok(!spy.called)
-
-    channel.on('*', spy)
-    channel.trigger('*', { type: 'INSERT' }, defaultRef)
+    channel.trigger('realtime', { event: 'INSERT' }, defaultRef)
     assert.ok(!spy.called)
 
-    channel.on('*', spy)
-    channel.trigger('INSERT', { type: 'INSERT' }, defaultRef)
+    channel.on('realtime', { event: 'INSERT' }, spy)
+    channel.trigger('realtime', { event: 'INSERT' }, defaultRef)
```

Comment:
Hey @w3b6x9 !

Is this work in progress, or are we removing `*` option from `.on()`?

Reply:
@dshukertjr ah, this is a work in progress so `*` still works:

e.g. `channel.on('realtime', { event: '*', schema:..., table:... }, spy)`

Reply:
Nice! I see the change now! Thanks @w3b6x9